packages/application-shell/src/components/setup-flop-flip-provider/setup-flop-flip-provider.tsx

```diff
@@ -52,6 +52,12 @@ type TParsedHttpAdapterFlags = Record<
 // is no need to be concerned about security.
 const ldClientSideIdProduction = '5979d95f6040390cd07b5e01';
 
+// NOTE: use this hardcoded value instead the one above,
+// to have the feature flag enabled in the playground
+// and to test the staging environment in the MC project main-navigation-test
+
+// const ldClientSideIdProduction = '5979d95f6040390cd07b5e00';
```

Comment:
In order to test the flags for local and staging we require to set manually this different LaunchDarkly ID.

Reply:
We should pass this via app config as an additional environment value to the playground app.
See how the internal apps are configured.

Reply:
Thanks for the heads-up! Done here - [bfba559](https://github.com/commercetools/merchant-center-application-kit/pull/3159/commits/bfba5594a733401b2f76c7c49c5e60f8becad17f).